--- taglib-1.9.1/bindings/c/CMakeLists.txt.orig	2013-08-02 18:13:26.000000000 -0500
+++ taglib-1.9.1/bindings/c/CMakeLists.txt	2013-08-02 18:13:37.000000000 -0500
@@ -31,21 +31,6 @@
 # themselves won't pull in the C++ libs -- and if a C application is
 # using the C bindings then we get link errors.
 check_library_exists(Crun __RTTI___ "" HAVE_CRUN_LIB)
-if(HAVE_CRUN_LIB)
-  # Which libraries to link depends critically on which
-  # STL version is going to be used by your application
-  # and which runtime is in use. While Crun is pretty much
-  # the only game in town, the three available STLs -- Cstd,
-  # stlport4 and stdcxx -- make this a mess. The KDE-Solaris
-  # team supports stdcxx (Apache RogueWave stdcxx 4.1.3).
-  
-  # According to http://bugs.kde.org/show_bug.cgi?id=215225 the library can have the following two names:
-  find_library(ROGUEWAVE_STDCXX_LIBRARY NAMES stdcxx4 stdcxx)
-  if(NOT ROGUEWAVE_STDCXX_LIBRARY)
-    message(FATAL_ERROR "Did not find supported STL library (tried stdcxx4 and stdcxx)")
-  endif()
-  target_link_libraries(tag_c ${ROGUEWAVE_STDCXX_LIBRARY} Crun)
-endif()
 
 set_target_properties(tag_c PROPERTIES
   VERSION 0.0.0
